dotnet/src/Connectors/Connectors.Memory.Sqlite/Conditions/SqliteWhereCondition.cs

```diff
@@ -15,6 +15,6 @@ internal abstract class SqliteWhereCondition(string operand, List<object> values
     public abstract string BuildQuery(List<string> parameterNames);
 
     protected string GetOperand() => !string.IsNullOrWhiteSpace(this.TableName) ?
-        $"{this.TableName}.{this.Operand}" :
-        this.Operand;
+        $"[{this.TableName}].[{this.Operand}]" :
```

Comment:
In SQLite, the standard quoting mechanism is double-quoted (the square brackets is more of a SQL Server thing). In fact I'm surprised this works!

Also, don't we need to escape TableName and Operand, in case they contain special characters?

I'd maybe suggest having a single method somewhere (RenderSqlIdentifier?) which does the quoting and escaping, and used from everywhere (for column, table, index names...).